static/app/views/performance/database/databaseSpanSummaryPage.tsx

```diff
@@ -89,6 +85,29 @@ export function DatabaseSpanSummaryPage({params}: Props) {
 
   const spanMetrics = data[0] ?? {};
 
+  const {
+    isLoading: isTransactionsListLoading,
+    data: transactionsList,
+    meta: transactionsListMeta,
+    error: transactionsListError,
+    pageLinks: transactionsListPageLinks,
+  } = useSpanMetrics({
+    search: MutableSearch.fromQueryObject(filters),
+    fields: [
+      'transaction',
+      'transaction.method',
+      'spm()',
+      `sum(${SpanMetricsField.SPAN_SELF_TIME})`,
+      `avg(${SpanMetricsField.SPAN_SELF_TIME})`,
+      'time_spent_percentage()',
+      `${SpanFunction.HTTP_ERROR_COUNT}()`,
+    ],
+    sorts: [sort],
+    limit: TRANSACTIONS_TABLE_ROW_COUNT,
+    cursor,
+    referrer: 'api.starfish.span-transaction-metrics',
```

Comment:
we should probably rename this referrer eventually

Reply:
Yes, definitely! Cleaning up referrers everywhere is on my list